docs/reference/technology/USB.md

```diff
@@ -46,7 +46,7 @@ void USBComponent::do_something_internal()
 
 ## USB device state
 
-USB defines 5 separate states a device can be in - Attached, Powered, Default, Address and Configured. Each state adds functionality. The Attached state has the least functionality, and the Configured state has the most functionality.
+USB defines five separate states of a device: **Attached**, **Powered**, **Default**, **Address** and **Configured**. Each state adds functionality. The **Attached** state has the least functionality, and the **Configured** state has the most functionality.
```

Comment:
I love the rewording, but why bold the states?

Reply:
Typically do that for introducing key words and names.